Tasks/app-store-promote/app-store-promote.ts

```diff
@@ -65,14 +71,28 @@ async function run() {
         // Install the ruby gem for fastlane
         tl.debug('Checking for ruby install...');
         tl.which('ruby', true);
-        // Install the fastlane tools (if they're already present, should be a no-op)
-        let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
-        gemRunner.arg(['install', 'fastlane']);
-        await gemRunner.exec();
-        // Always update fastlane (if already latest, should be a no-op)
-        gemRunner = tl.tool(tl.which('gem', true));
-        gemRunner.arg(['update', 'fastlane', '-i', gemCache]);
-        await gemRunner.exec();
+
+        // If desired, install the fastlane tools (if they're already present, should be a no-op)
+        if (installFastlane) {
+            tl.debug('Installing fastlane...');
+            let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
+            gemRunner.arg(['install', 'fastlane']);
+            if (fastlaneVersionToInstall) {
+                tl.debug(`Installing specific version of fastlane: ${fastlaneVersionToInstall}`);
+                gemRunner.arg(['-v', fastlaneVersionToInstall]);
```

Comment:
What if I already have v1 installed, and I want to install v2?  Would this 'gem install  fastlane -v 2' run and overwrite old fastlane v1?